sim/scenarios/nat/README.md

```diff
@@ -0,0 +1,24 @@
+# NAT Rebinding
+
+This scenario uses a bottleneck link similar to the [simple-p2p](../simple-p2p)
+scenario and simulates a NAT that is frequently changing the source addresses of
+all client connections. This scenario can be used to test whether QUIC
+implementations correctly handle NAT and CGN rebindings (in the latter case, the
+IP address also changes and not just the port.) All flows are rebound to new
+source ports (and possibly IP a new IP address) every time the timer expires.
+
+This scenario has the following configurable properties:
+
+* `--first_rebind`: The time of the first NAT/CGN rebind event. Zero to disable.
+  For example, `--first_rebind=1s`.
```

Comment:
Why treat 0 special here? Disabling doesn't make a lot of sense here, does it? If people don't want NAT rebinding, they can just use simple-p2p.

Reply:
I would require this to be > 0.